components/02-elements/cart-message/_cart-message.scss

```diff
@@ -0,0 +1,56 @@
+$cart-message-background        : $gray;
```

Comment:
Naming is wrong, it's not just a "cart message", it's for all kind of notifications, even when you open M code, you will see it's named "messages".